contrib/install_db4.sh

```diff
@@ -1,26 +1,21 @@
 #!/bin/sh
 
-# Install libdb4.8 (Berkeley DB).
+# This script downloads, patches, configures and builds Berkeley DB 4.8
+# and installes it in a given directory. This enables Bitcoin Core
+# to be built with --enable-wallet
 
 set -e
 
-if [ -z "${1}" ]; then
-  echo "Usage: ./install_db4.sh <base-dir> [<extra-bdb-configure-flag> ...]"
-  echo
-  echo "Must specify a single argument: the directory in which db4 will be built."
-  echo "This is probably \`pwd\` if you're at the root of the bitcoin repository."
-  exit 1
-fi
+DBVERS="db-4.8.30.NC"
+DBFILE="$DBVERS.tar.gz"
+DBPAGE="http://download.oracle.com/berkeley-db/$DBFILE"
+DBHASH='12edc0df75bf9abd7f82f821795bcee50f42cb2e5f76a6a281b85732798364ef'
 
-expand_path() {
-  echo "$(cd "${1}" && pwd -P)"
+err() {
+	echo $@ >&2
```

Comment:
As a code convention to the rest of this file I think you must remove tabs and keep two spaces indentation here.